cmd/wallet-web/src/components/Vaults/VaultCard.vue

```diff
@@ -1,6 +1,7 @@
 <template>
   <div>
```

Comment:
Vue 3 allows us to have [multiple root-level elements ](https://v3.vuejs.org/guide/migration/fragments.html#_2-x-syntax), so we should not need this `<div>` here

Reply:
removed